### PR TITLE
Awesome Books With ES6

### PR DESCRIPTION
**Pull request by mwismann**
### Awesome Books With ES6  😎 
**In this branch I implemented the following features:**

- An entry point for your JavaScript code called index.js ✅ 
- index.css file for styles ✅ 
- A directory called modules ✅ 
- ES6 syntax ✅ 
- A separate file for each module in the modules directory ✅ 
- Kept all functionalities of the app without errors ✅ 
- Current date below the navigation bar with luxon library ✅

### CODE REVIEW

File: modules/classes.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    this.title = title;
+    this.author = author;
+  }
+
+    static books = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
+
+    addBook() {
+      if (newBookTitle.value.trim() === '' || newBookAuthor.value.trim() === '') {
+        return;
+      }
```
Comment (nahnahdev) [inline modules/classes.js:17]: - [ ] Great job so far :+1: According to the project requirements, you are to refactor your functions to arrow functions but that hasn't been implemented in this part of your codebase.
- [ ] You can read more about [Why arrow functions are used in Es6 here](https://www.geeksforgeeks.org/arrow-functions-in-javascript/#:~:text=this%20and%20Arrow%20Functions%3A%20Arrow,is%20lexically%20or%20statically%20defined.). So kindly include that requirement in your project :wink: 
- Project Requirement
image

File: modules/classes.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          `<article class="book-item">
+            <div class="item-meta">
+              <h2>"${book.title}"</h2>
+              <span>by</span>
+              <p>${book.author}</p>
+            </div>
+              <button class="remove-book btn" data-id="${book.id}">Remove</button>
+            </article>`);
+      });
+
```
Comment (nahnahdev) [inline modules/classes.js:48]: - [ ] Kindly refactor these functions into arrow functions as well :wink:

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @mwismann , 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Exceptional Project Documentation :heavy_check_mark: 
- Linters well installed and passing :heavy_check_mark: 
- Great job integrating Luxon in your project :heavy_check_mark: 
- Good use of modules in your project :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @mwismann 👋,

_This is an additional review as a result of the second opinion request._

- Class methods are ES6 syntax and should not be refactored to arrow functions for proper implementation and functionality of the `this`, `constructor` and `super` keywords in classes.
@NahnahAJ, we only required students to refactor functions to arrow functions and not class methods.


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)


Keep rocking 🚀!

**Review by ShadyShawkat — APPROVED** [review]
## Hi @mwismann, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**